.github/workflows/scheduled.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     env:
       CI: 1
       HOMEBREW_COLOR: 1
-      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_ANALYTICS_THIS_RUN: 1
```

Comment:
This isn't meant to be a user-accessible variable (it's not documented). Could `HOMEBREW_NO_ANALYTICS` be set/unset more specifically where desired?